src/pages/version/Metadata.tsx

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+import { MetadataCard } from "components/MetadataCard";
+import { StyledLink } from "components/styles";
+import { P2 } from "components/Typography";
+import { getCommitQueueRoute, getProjectPatchesRoute } from "constants/routes";
+import { environmentalVariables, string } from "utils";
+import { ParametersModal } from "./ParametersModal";
+
+const { msToDuration, getDateCopy } = string;
+const { getUiUrl } = environmentalVariables;
+
+interface Props {
+  loading: boolean;
+  version: {
+    project: string;
+    author: string;
+    revision: string;
+    createTime: Date;
+    startTime?: Date;
+    finishTime?: Date;
+    projectIdentifier: string;
+    baseVersionID?: string;
+    versionTiming?: {
+      makespan?: number;
+      timeTaken?: number;
+    };
+    parameters: {
+      key: string;
+      value: string;
+    }[];
+    commitQueuePosition?: number;
+    isPatch: boolean;
+  };
+}
+
+export const Metadata: React.FC<Props> = ({ loading, version }) => {
+  const {
+    author,
+    revision,
+    project,
+    versionTiming,
+    createTime,
+    startTime,
+    finishTime,
+    commitQueuePosition,
+    projectIdentifier,
+    baseVersionID,
+    isPatch,
+    parameters,
+  } = version || {};
+  const { makespan, timeTaken } = versionTiming || {};
+  return (
+    <MetadataCard
+      loading={loading}
+      error={null}
+      title={isPatch ? "Patch Metadata" : "Version Metadata"}
+    >
+      <P2>
+        Project:{" "}
+        <StyledLink href={getProjectPatchesRoute(project)}>
+          {projectIdentifier}
+        </StyledLink>
+      </P2>
+      <P2>Makespan: {makespan && msToDuration(makespan)}</P2>
+      <P2>Time taken: {timeTaken && msToDuration(timeTaken)}</P2>
+      <P2>Submitted at: {getDateCopy(createTime)}</P2>
+      <P2>Started: {startTime && getDateCopy(startTime)}</P2>
+      <P2>Finished: {finishTime && getDateCopy(finishTime)}</P2>
+      <P2>{`Submitted by: ${author}`}</P2>
+      {baseVersionID && revision && (
+        <P2>
+          <StyledLink
```

Comment:
this could also be a StyledRouterLink

Reply:
This links to a legacy ui page so needs to be a normal link

Reply:
ah right, I think it could be possible to link this to the spruce version page now since it goes live in this PR.